styles/lefty/themes.s2

```diff
@@ -1,3 +1,325 @@
+#NEWLAYER: lefty/beachhut
+layerinfo type = "theme";
+layerinfo name = "Beach Hut";
+layerinfo redist_uniq = "lefty/beachhut";
+layerinfo author_name = "forthwritten";
+
+set theme_authors = [ { "name" => "forthwritten", "type" => "user" } ];
+
+##===============================
+## Page Colors
+##===============================
+
+set color_page_background = "#c2f2f2";
+set color_page_left_border = "#e7ce8c";
+set color_page_link = "#da284c";
+set color_page_link_active = "#ff8279";
+set color_page_link_hover = "#ff8279";
+set color_page_link_visited = "#da284c";
+set color_page_navigation_link = "#da284c";
+set color_page_navigation_link_active = "#ff8279";
+set color_page_navigation_link_hover = "#ff8279";
+set color_page_navigation_link_visited = "#da284c";
+set color_page_right_border = "#da284c";
+set color_page_title = "#272727";
+set color_header_background = "#f34384";
+set color_header_link = "#fff";
+set color_header_link_active = "#fff";
+set color_header_link_hover = "#fff";
+set color_header_link_visited = "#fff";
+set color_header_title_background = "#e7ce8c";
+set color_headernav_background = "#da284c";
+set color_headernav_current_background = "#f34384";
+set color_headernav_hover_background = "#f34384";
+set color_footer_background = "#fff";
+set color_footer_link = "#da284c";
+set color_footer_link_active = "#ff8279";
+set color_footer_link_hover = "#ff8279";
+set color_footer_link_visited = "#da284c";
+
+##===============================
+## Entry Colors
+##===============================
+
+set color_entry_background = "#fff";
+set color_entry_interaction_links = "#da284c";
+set color_entry_interaction_links_active = "#ff8279";
+set color_entry_interaction_links_hover = "#ff8279";
+set color_entry_interaction_links_visited = "#da284c";
+set color_entry_link = "#da284c";
+set color_entry_link_active = "#ff8279";
+set color_entry_link_hover = "#ff8279";
+set color_entry_link_visited = "#da284c";
+set color_entry_text = "#272727";
+set color_entry_title = "#da284c";
+set color_entry_title_background = "#fff";
+set color_entry_titleuserpic_border = "#e7ce8c";
+set color_comment_footer_text = "#da284c";
+set color_comment_title = "#da284c";
+
+##===============================
+## Module Colors
+##===============================
+
+set color_module_link = "#da284c";
+set color_module_link_active = "#ff8279";
+set color_module_link_hover = "#ff8279";
+set color_module_link_visited = "#da284c";
+set color_module_text = "#da284c";
+set color_module_title = "#da284c";
+
+##===============================
+## Fonts
+##===============================
+
+set font_journal_title = "arial"; 
```

Comment:
It works in my Linux distro! But it's a quick fix anyway.

On Thu, Jun 25, 2015 at 12:01 AM, Afuna <notifications@github.com> wrote:

> In styles/lefty/themes.s2
> <https://github.com/dreamwidth/dw-free/pull/1437#discussion_r33220644>:
>
> > +##===============================
> > +## Module Colors
> > +##===============================
> > +
> > +set color_module_link = "#da284c";
> > +set color_module_link_active = "#ff8279";
> > +set color_module_link_hover = "#ff8279";
> > +set color_module_link_visited = "#da284c";
> > +set color_module_text = "#da284c";
> > +set color_module_title = "#da284c";
> > +
> > +##===============================
> > +## Fonts
> > +##===============================
> > +
> > +set font_journal_title = "arial";
>
> Note: this needs to be capitalized, because otherwise this won't show up
> as arial in some OS (linux I think?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dreamwidth/dw-free/pull/1437/files#r33220644>.
>